tests/ui/invalid-compile-flags/crate-type-flag.rs

```diff
@@ -17,12 +17,12 @@
 //@[staticlib] compile-flags: --crate-type=staticlib
 //@[staticlib] check-pass
 
-//@[dylib] ignore-musl (dylibs are not supported)
+//@[dylib] ignore-musl (dylibs are supported, but target is statically linked by default)
 //@[dylib] ignore-wasm (dylibs are not supported)
 //@[dylib] compile-flags: --crate-type=dylib
 //@[dylib] check-pass
 
-//@[cdylib] ignore-musl (cdylibs are not supported)
+//@[cdylib] ignore-musl (dylibs are supported, but target is statically linked by default)
```

Comment:
Nit: should be "dynamic linking is supported but musl libc is statically linked by default"